extensions/ql-vscode/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # CodeQL for Visual Studio Code: Changelog
 
-## [UNRELEASED]
+## 1.3.7 - 24 November 2020
```

Comment:
Is now the right to create the next UNRELEASED section? Or is that done by the bump?

Reply:
It's done by the bump.